ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -998,6 +1053,13 @@ for target in "${TARGET_RELEASES[@]}"; do
     fi
     check_history
     health_check
+    currentVersion=$(oc version -o yaml | grep openshiftVersion | grep -o '[0-9]*[.][0-9]*' | head -1)
+    currentPlugin=$(oc get network.config.openshift.io cluster -o jsonpath='{.status.networkType}')
+    if [[ ${currentVersion} == "4.16" && ${currentPlugin} == "OpenShiftSDN" ]]; then
+	echo "the cluster is 4.16 and openshift-sdn need to migration to ovn and then upgrade if need"
```

Comment:
Suggest to use "The cluster is running version 4.16 with OpenShift SDN, and it needs to be migrated to OVN before upgrading"

Reply:
updated, thanks.